cmd/start.go

```diff
@@ -15,13 +15,16 @@ const (
 	serviceProviderFlagName = "service-provider"
 	socketPathFlagName      = "socket-path"
 	syncProviderFlagName    = "sync-provider"
+	providerArgsFlagName    = "sync-provider-args"
 	evaluatorFlagName       = "evaluator"
 	serverCertPathFlagName  = "server-cert-path"
 	serverKeyPathFlagName   = "server-key-path"
 	uriFlagName             = "uri"
 	bearerTokenFlagName     = "bearer-token"
 )
 
+var ProviderArgs map[string]string
```

Comment:
`ProviderArgs` is also defined in pkg/sync, this one doesn't seem to be used, within cmd the argument seems to just be referenced as a `map[string]string` with viper

Reply:
Removed